cdap-data-fabric/src/main/resources/cdap/grok/patterns/firewalls

```diff
@@ -0,0 +1,60 @@
+# NetScreen firewall logs
```

Comment:
Do we need license header for all these pattern files? If not, why? As far as I know, it is distributed as part of the software.